etc/toolbox/interface_browser.tool

```diff
@@ -0,0 +1,4 @@
+name=Interface browser
+uri=/interface_browser/
+icon=/static/images/toolbox/info.png
```

Comment:
The tool needs a unique icon.

My immediate suggestion would be a port graphic, but this is already in use by the Port Admin tool.

My second suggestion is to make a new icon based on the port admin icon, with several overlapping ports (to sort of indicate this tool shows many ports)